GPU/Common/FramebufferCommon.cpp

```diff
@@ -40,16 +40,44 @@ void CenterRect(float *x, float *y, float *w, float *h, float origW, float origH
 		outW = frameW;
 		outH = frameH;
 	} else {
-		// Add special case for 1080p displays, cutting off the bottom and top 1-pixel rows from the original 480x272.
-		// This will be what 99.9% of users want.
-		if (origW == 480 && origH == 272 && frameW == 1920 && frameH == 1080 && !rotated) {
-			*x = 0;
-			*y = -4;
-			*w = 1920;
-			*h = 1088;
-			return;
+		bool fullScreenZoom = true;
+#ifdef _WIN32
```

Comment:
Oh yeah I get the fullscreen idea by looking at the config. Will change the config one as well then.

 CenterRect was always like that, I can just change it everywhere I guess, I just wonder about this https://github.com/hrydgard/ppsspp/blob/c2f4fad56aada331bbd8dd7db99037285bab853f/GPU/Software/SoftGpu.cpp#L52 is this code even used? Because that feature I added seems to work also in software mode.